styles.css

```diff
@@ -0,0 +1,61 @@
+@import url('https://fonts.googleapis.com/css2?family=BhuTuka+Expanded+One&family=Cormorant+Garamond:wght@300&family=Dancing+Script:wght@500;600;700&family=Mukta:wght@200&family=Permanent+Marker&family=Redressed&family=Roboto:wght@100&family=Signika:wght@300;400;500;600;700&family=Space+Grotesk:wght@300&display=swap');
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 95px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+.nav h3 {
+  margin-left: 15px;
+}
+
+.nav i {
+  margin-right: 25px;
+}
+
+.main {
+  justify-content: center;
+  padding: 15px 15px;
+  color: white;
+  background-color: #1c1a19;
+  background-image: url(images/bg.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
- [ ] The background image for this style is too large and is making the text not visible. It's meant to be small and placed at the bottom right corner just as it's in the Figma design. Kindly style your app to match the design given. 
- [ ] On this same screenshot, please move the social media icons to the left a bit as directed by the arrow on the screenshot.
- [ ] Lastly on the same screenshot, I noticed there is a margin on all four sides of the screen. The reason for that is because you are not making use of reset styles.
- Use the code below to add basic reset styles to your page. Place it at the top of your CSS styles
```mdx
* {
  margin: 0;
  padding: 0;
}
```
**See Your Design**

![image](https://user-images.githubusercontent.com/50941074/170681256-521741ac-d294-4dbb-a93f-ae8b1a714c1a.png)

**The Original Design**

![image](https://user-images.githubusercontent.com/50941074/170681467-fcc2eecb-b45a-4ba4-86f8-14ed03ccbdd8.png)